src/BundledBindings/InputBinding.cs

```diff
@@ -17,32 +17,29 @@ public class InputBinding : IInputBinding
         public override BindingInformation? ExtractBinding(AttributeAst attribute, ParameterAst parameter)
         {
             BindingInformation bindingInformation = new BindingInformation();
-            string? bindingType = WorkerIndexingHelper.GetPositionalArgumentStringValue(attribute, 0);
-            string? bindingName = WorkerIndexingHelper.GetPositionalArgumentStringValue(attribute, 1);
+            bindingInformation.Direction = BindingDirection;
+
+            string bindingType = WorkerIndexingHelper.GetNamedArgumentStringValue(attribute, "Type");
+            string bindingName = WorkerIndexingHelper.GetNamedArgumentStringValue(attribute, "Name");
 
             List<string> problems = new List<string>();
 
-            if (string.IsNullOrEmpty(bindingType))
+            if (string.IsNullOrWhiteSpace(bindingType))
             {
                 problems.Add(AzPowerShellSdkStrings.MissingType);
             }
-            if (string.IsNullOrEmpty(bindingName))
+            if (string.IsNullOrWhiteSpace(bindingName))
             {
                 bindingName = parameter.Name.VariablePath.UserPath;
             }
 
-            bindingInformation.Direction = BindingDirection;
-
             if (problems.Count > 0)
             {
                 throw new Exception(AzPowerShellSdkStrings.InputBindingProblemsExist + string.Join("\n", problems));
```

Comment:
This value "\n" might need to be "`n" for a new line in PowerShell.

Reply:
Since it's written in C#, that gets translated to newline before it's passed over to PowerShell. As far as I can tell, the reason we're seeing the text \n in the error message is because PowerShell does not support newlines in error messages, so it prints the text value of the character to let you know that it is there.

Reply:
Yes you are correct. Let's leave it as is for now, and we can come back to this after the bug bash.